decidim-consultations/app/views/decidim/consultations/admin/consultations/results.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="card" id="consultations">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= t "decidim.admin.titles.results" %>
+      <span class="label button--title"><%= t "decidim.admin.consultations.results.total_votes", count: current_consultation.total_votes %></span>
+    </h2>
+  </div>
+  <div class="card-section">
+
+    <% current_consultation.questions.published.each do |question| %>
```

Comment:
I think we should add `include` to avoid N+1 queries:

```suggestion
    <% current_consultation.questions.published.include(:responses).each do |question| %>
```

Reply:
indeed, thanks